code/game/gamemodes/game_mode.dm

```diff
@@ -609,7 +609,7 @@
 	var/datum/station_goal/pizza_delivery/papa = new
 	station_goals += papa
 	return
-
+	/*

```

Comment:
Papa Johns-related change.

Reply:
Whadaya want me to do about it

Reply:
Just pointing it out for the downstreams @qwertyquerty